spk/pyload/Makefile

```diff
@@ -0,0 +1,45 @@
+SPK_NAME = pyload
+SPK_VERS = 0.4.9
+SPK_REV = 1
+SPK_ICON = src/pyload.png
+DSM_UI_DIR = app
+
+DEPENDS = cross/tesseract-ocr cross/tessdata-eng cross/js cross/$(SPK_NAME)
+SPK_DEPENDS = "python>2.7.3-3"
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = pyLoad is a fast, lightweight and full featured download manager for many One-Click-Hoster, container formats like DLC, video sites or just plain http/ftp links.
+ADMIN_PORT = 8190
+RELOAD_UI = yes
+STARTABLE = yes
+DISPLAY_NAME = pyLoad
+
+HOMEPAGE   = http://pyload.org
+LICENSE    = GPLv3
+
+WIZARDS_DIR = src/wizard/
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
+FWPORTS          = src/${SPK_NAME}.sc
+
+INSTALL_PREFIX = /usr/local/$(SPK_NAME)
+
+POST_STRIP_TARGET = pyload_extra_install
+
+include ../../mk/spksrc.spk.mk
+
+.PHONY: pyload_extra_install
+pyload_extra_install:
+	install -m 755 -d $(STAGING_DIR)/app
+	install -m 644 src/app/config $(STAGING_DIR)/app/config
+	install -m 755 -d $(STAGING_DIR)/etc
+	install -m 644 src/default_syn.conf $(STAGING_DIR)/etc/pyload.conf
```

Comment:
Forgot this one: Probably rename this file to `pyload.conf`, instead of `default_syn.conf`